app/components/register.jsx

```diff
@@ -53,29 +56,67 @@ export class Register extends Component {
         <div className="register-container">
           <NavComponent />
           <ReloadComponent />
-          <div className="form">
+          <div className="form register">
             <h1>Register</h1>
             <form onSubmit={this.register}>
               <AlertComponent message={error} type="error" />
+
+              <h2>Account Info</h2>
               <div className="form-group">
                 <label htmlFor="username">Username</label>
-                <input ref={(c) => this._username = c} name="username" id="username" type="text" required placeholder="ashketchum10" autoComplete="off" autoCorrect="off" autoCapitalize="off" spellCheck="false" />
+                <input className="form-control" ref={(c) => this._username = c} name="username" id="username" type="text" required placeholder="ashketchum10" autoComplete="off" autoCorrect="off" autoCapitalize="off" spellCheck="false" />
                 <i className="fa fa-asterisk" />
               </div>
               <div className="form-group">
                 <label htmlFor="password">Password</label>
-                <input ref={(c) => this._password = c} name="password" id="password" type="password" required placeholder="••••••••••••" />
+                <input className="form-control" ref={(c) => this._password = c} name="password" id="password" type="password" required placeholder="••••••••••••" />
                 <i className="fa fa-asterisk" />
               </div>
               <div className="form-group">
                 <label htmlFor="password_confirm">Confirm Password</label>
-                <input ref={(c) => this._password_confirm = c} name="password_confirm" id="password_confirm" type="password" required placeholder="••••••••••••" />
+                <input className="form-control" ref={(c) => this._password_confirm = c} name="password_confirm" id="password_confirm" type="password" required placeholder="••••••••••••" />
                 <i className="fa fa-asterisk" />
               </div>
               <div className="form-group">
                 <label htmlFor="friend_code">Friend Code</label>
-                <input ref={(c) => this._friend_code = c} name="friend_code" id="friend_code" type="text" placeholder="XXXX-XXXX-XXXX" onChange={(e) => this._friend_code.value = friendCode(e.target.value)} />
+                <input className="form-control" ref={(c) => this._friend_code = c} name="friend_code" id="friend_code" type="text" placeholder="XXXX-XXXX-XXXX" onChange={(e) => this._friend_code.value = friendCode(e.target.value)} />
+              </div>
+
+              <h2>
+                First Dex Info
+                <div className="tooltip">
+                  <i className="fa fa-plus-circle" />
```

Comment:
i think i just ported it over from the tooltips we have already on the tracker page:
![image](https://cloud.githubusercontent.com/assets/5498072/20235926/e6be841c-a856-11e6-808a-d279f61da888.png)

but in that context, the `+` makes sense while it doesn't rly here. will change to a `?`!